### PR TITLE
Require 90% coverage on all categories of jest coverage

### LINKED ISSUE #37: Add code coverage check to unit tests
As an engineer, I want the unit test CI pipeline check to also verify that code is adequately covered by unit tests, with a 90% threshold (minimum) for all categories.

Depends on #36, and unit tests being created for all the components.

Recommend opening a PR and having folks assign themselves a component or two to add test coverage to, until all components have sufficient test coverage

### PR DESCRIPTION
**Pull request by kleinschmidtj**
## Resolves #37 

Changes proposed in this pull request:

- Adds test coverage requirement of 90% in CI

### CODE REVIEW

File: jest.config.ts [hunk 1]
```diff
@@ -8,6 +8,14 @@ const config: Config = {
...
   ],
   coverageDirectory: 'coverage/jest',
+  coverageThreshold: {
+    global: {
+      statements: 90,
+      branches: 90,
+      functions: 90,
+      lines: 90,
+    },
+  },
```
Comment (brandonlenz) [inline jest.config.ts:18]: Alright, we have a target now 😄

**Review by brandonlenz — APPROVED** [review]

File: src/components/form/fields/CheckboxGroupField/CheckBoxGroupField.test.tsx
[the remark this answers is not in the document]
Comment (brandonlenz) [reply src/components/form/fields/CheckboxGroupField/CheckBoxGroupField.test.tsx:1]: https://github.com/trussworks/unemployment-insurance-modernization-demo/issues/120
